docs/docs/Release_Notes/2.0.0/2.0.0_Breaking_Changes.md

```diff
@@ -0,0 +1,70 @@
+# Breaking Changes - GA (v.2.0.0)
+
+## Plugin Config Field Renamed
+The following field in the [Plugin Config](/References/Plugin_Config/) were replaced:
+
+| Previous | Updated |
+| -------- | ------- |
+| `version` | `buildNumber` |
+
+Additionally, the `buildNumber` must be a string that conforms to the following rules:
+
+* The string must be composed of a sequence of non-negative integers, not all zero, separated by periods.
+* Trailing zeros are ignored. So, "1.0.0" is treated the same as "1".
+* Build numbers are sortable numerically, with earlier numbers having more significance than later numbers. So, "2.0" comes after "1.99999", and "1.10" comes after "1.2".
+* The Delphix Engine will never allow installation of plugin with a build number that is ordered before the the already-installed build number.
+
+
+More details about the format are [here](/Versioning_And_Upgrade/Versioning.md#build-number-format-rules).
+
+### What is affected
+All plugins built with v1.0.0 or below will be affected. The [Plugin Config](/References/Plugin_Config) field `version` will have to be updated to `buildNumber`.
+
+### How does it fail
+[dvp build](/References/CLI.md#build) will fail with the following error message if the [Plugin Config](/References/Plugin_Config) `version` field is not updated to `buildNumber`:
```

Comment:
We also added a wait option to dvp upload. So may be Jeff can provide those details if we want to document that?

Reply:
good point, will add

Reply:
done